hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/FileSizeCountTask.java

```diff
@@ -138,11 +145,17 @@ public Collection<String> getTaskTables() {
    * Read the Keys from update events and update the count of files
    * pertaining to a certain upper bound.
    *
-   * @param events Update events - PUT/DELETE.
-   * @return Pair
+   * @param events            The batch of OM update events to be processed.
+   * @param subTaskSeekPosMap A map containing the seek positions for
+   *                          each sub-task, indicating where to start processing.
+   * @return A {@link TaskResult} containing:
+   *         - The task name.
+   *         - A map of sub-task names to their respective seek positions.
+   *         - A boolean indicating whether the task was successful.
    */
   @Override
-  public Pair<String, Boolean> process(OMUpdateEventBatch events) {
+  public TaskResult process(OMUpdateEventBatch events,
+                            Map<String, Integer> subTaskSeekPosMap) {
```

Comment:
We aren't doing anything with subTaskSeekPosMap here in this Class?

Reply:
No, current PR is only for NSSummary task, later the same interface can be extended for tracking the event iterator position for other tasks as well in future PRs if any.